kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunner.kt

```diff
@@ -50,19 +49,22 @@ interface WorkflowRunner<out OutputT> {
 
   /**
    * A stream of the [output][OutputT] values emitted by the running [Workflow].
+   * Outputs emitted while there are no active observers are cached -- that is,
+   * all outputs will be delivered to some observer.
    */
-  val output: Flowable<out OutputT>
+  val output: LiveData<out OutputT>
```

Comment:
Does AndroidX have helpers to observe publishers with a lifecycle? If it does, maybe we should just expose outputs as a Publisher.

Reply:
Oh, I like that a lot. Will check.

Reply:
I don't think it does, I checked too.

Reply:
In fact, realizing that LiveData offers us no value. Going to make Publisher our baseline.

Reply:
> I don't think it does, I checked too.

It has `fun Publisher.fromPublisher(): LiveData`, and that's all you need. Lifecycle gets applied at subscribe time, it's not built in to the LiveData source itself.

Regardless, I won't merge this PR until I'm confident we've solved the outputs race.